bindata/network/multus/004-configmap.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cni-copy-bash-script
+  namespace: openshift-multus
+  labels:
+    app: openshift-multus
+data:
+  copycni.sh: |-
+    #!/bin/bash
+    set -x
+    # source our os-release variables.
+    if [ -f "/host/etc-os-release/os-release" ]; then
```

Comment:
So rather than all of this, what about just `uname -r`? If it contains `.el7` you're on RHEL 7 and if it contains `.el8` you're on RHEL 8.

Reply:
@dougbtv fwiw, on fedora `/etc/os-release` is a symlink to `/usr/lib/os-release` but on rhel8 and rhel7, they don't seem to be symlinks, atleast on my envs, but anyway `uname -r` looks great.

Reply:
> So rather than all of this, what about just uname -r? If it contains .el7 you're on RHEL 7 and if it contains .el8 you're on RHEL 8.

Mrm...yes but `/etc/os-release` is *designed* for this use case...parsing kernel version strings isn't.

Reply:
Hard-coding knowledge of where RHEL points its symlinks to is gross. But given Casey's comment below about using a `type: File` mount, we can just parse `/etc/os-release`.

Reply:
I kept the methodology to use the `/etc/os-release`, and have now implemented Casey's suggesiton on the `type: File` so I could remove the double dip for the file + symlink.